Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:alpine 
+COPY ./ /opt 
+WORKDIR "/opt" 
+RUN npm cache clean --force && rm -rf node_modules && npm install 
+
+EXPOSE 3000
+
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.5.0/wait /wait
```

Comment:
Can you explain the use case of this here

Reply:
https://docs.docker.com/compose/startup-order/

Reply:
You're using this but not passing any arguments during it's execution, then is it really necessary? Or else you can pass the postgres port in the execution arguments for it to work as expected

Reply:
In the `Dockerfile`, we're waiting for the `postgres` instance to spin up:
```yml
environment:
      WAIT_HOSTS: postgres:5432
```